src/components/config/config.ts

```diff
@@ -90,8 +91,15 @@ export async function setActiveKubeconfig(kubeconfig: string): Promise<void> {
     await addPathToConfig(KUBECONFIG_PATH_KEY, kubeconfig);
 }
 
-export function getActiveKubeconfig(): string {
-    return vscode.workspace.getConfiguration(EXTENSION_CONFIG_KEY)[KUBECONFIG_PATH_KEY];
+export function getKubeconfig(): string {
+	let kubeConfig = vscode.workspace.getConfiguration(EXTENSION_CONFIG_KEY)[KUBECONFIG_PATH_KEY];
```

Comment:
>	 [](start = 0, length = 1)

Nit: it looks like the convention in this repo is to use spaces rather than tabs. Could you please replace these by spaces for consistency?

Reply:
Yes!